searchlib/src/tests/features/prod_features.cpp

```diff
@@ -410,8 +411,8 @@ Test::setupForAttributeTest(FtFeatureTest &ft, bool setup_env)
             .addField(FieldType::ATTRIBUTE, CollectionType::SINGLE, "slong")
             .addField(FieldType::ATTRIBUTE, CollectionType::SINGLE, "sdouble")
             .addField(FieldType::ATTRIBUTE, CollectionType::SINGLE, "sbyte")
-            .addField(FieldType::ATTRIBUTE, CollectionType::SINGLE, "sbool")
-            .addField(FieldType::ATTRIBUTE, CollectionType::SINGLE, "sebool");
+            .addField(FieldType::ATTRIBUTE, CollectionType::SINGLE, DataType::BOOL,"sbool")
```

Comment:
in the future: maybe we need to enforce passing the data type

Reply:
Agree.